Sources/TuistCore/Models/InfoPlist.swift

```diff
@@ -44,6 +44,7 @@ public enum InfoPlist: Equatable {
     }
 
     case file(path: AbsolutePath)
+    case generatedFile(path: AbsolutePath)
```

Comment:
Out of curiosity, why are we making a distinction between `file` and `generatedFile`. Do you think we'll have a use case where we might want to check if the Info.plist is a generated one?

Reply:
Ah this was a fun one :)

So we have a lint rule to validate an `infoPlist.file` exists on disk. The first time we generate a project the file won't and that causes the generation to fail! I thought perhaps adding another type `.generatedFile` which can bypass that lint check (this of course won't be exposed in manifests, purely internal).